source/utils/fs-async.js

```diff
@@ -0,0 +1,9 @@
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+fs.isFileExists = function(file) {
+  return fs.accessAsync(file, fs.F_OK)
```

Comment:
Node.js 0.10 does not support [`access`](https://github.com/nodejs/node/commit/165b70f146e163b82a09bb869463708516c08cf6). Maybe that's why [a lot of unit tests fail](https://travis-ci.org/FredrikNoren/ungit/jobs/99651029#L1018)?